draft-thoji-scone-trone-protocol.md

```diff
@@ -250,38 +250,44 @@ The Rate Signal field in TRONE uses the low 6 bits (0x3f) of the first byte.
 This field is encoded as a logarithmically spaced distribution over a range
 defined by the TRONE protocol version.
 
-The Rate Signal field is set to 0x3F (63) when sent by a QUIC endpoint,
-indicating no rate limit is in place or that the TRONE protocol is not supported
-by network elements on the path. Vales 0x00 through 0x3E (62) represent the ceiling
-of rate being advised by the network element(s) on the path.
+When sent by a QUIC endpoint, the Version field of a TRONE packet is set to
+0xTRONE2 and the Rate Signal field is set to 0x3F (63), indicating no rate limit
+is in place or that the TRONE protocol is not supported by network elements on
+the path. All other values (0x00 through 0x3F for protocol version 0xTRONE1 and
+0x00 through 0x3E for protocol version 0xTRONE2) represent the ceiling of rates
+being advised by the network element(s) on the path.
 
 For TRONE protocol version 0xTRONE1, the rate limits use a logarithmic scale with:
+
 * Base rate (b_min) = 100 Kbps
-* Maximum rate ≈ 125 Mbps
 * Bitrate at value n = b_min * 10^(n/20)
 
 For TRONE protocol version 0xTRONE2, the rate limits use a logarithmic scale with:
-* Base rate (b_min) = 10000 Kbps (10 Mbps)
-* Maximum rate ≈ 12.5 Gbps
-* Bitrate at value n = b_min * 10^(n/20)
 
-This creates two overlapping ranges:
-* Version 0xTRONE1: 100 Kbps to 125 Mbps
-* Version 0xTRONE2: 10 Mbps to 12.5 Gbps
+* Bitrate at value n = b_min * 10^((n + 64)/20)
+
+With two versions combined, bitrates between 100 Kbps and 178 Gbps can be
```

Comment:
178 is at n = 61 
n = 62 should be ~199.53. 

```suggestion
With two versions combined, bitrates between 100 Kbps and 199.5 Gbps can be
```

Reply:
Thank you!